libs/oidc/admin/ngx/src/lib/modules/clients/pages/view-client/view-client.module.ts

```diff
@@ -0,0 +1,25 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Routes, RouterModule } from '@angular/router';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+import { UIFormsModule } from '@tamu-gisc/ui-kits/ngx/forms';
+import { UILayoutModule } from '@tamu-gisc/ui-kits/ngx/layout';
+
+import { ViewClientComponent } from './view-client.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: ViewClientComponent,
+    pathMatch: 'full'
+  }
+];
+
+@NgModule({
+  imports: [CommonModule, RouterModule.forChild(routes), FormsModule, ReactiveFormsModule, UIFormsModule, UILayoutModule],
```

Comment:
You get exactly this much slack:


|----|


You'll find out pretty quickly if you need a module in a component, especially because you typically import them when you need a directive or component. Since these empty modules/components don't have anything, it stands to reason they don't need all FormModule, ReactiveFormsModule, UIFormsModuke, and UILayoutModule

Reply:
But don't children inherit from the parent? So if a child component requires FormModule won't the parent need it? Or no?

Also templates were *always* angry, so that was partially why I did it.

Reply:
> But don't children inherit from the parent? So if a child component requires FormModule won't the parent need it? Or no?


Only reason for imports in a module is for exported members by that module which typically only include directives and component tags (e.g. `formGroup`, `tamu-gisc-accordion`) and optionally any scoped providers. The only thing a component in this module had was a simple `<p>edit-client works!</p>` so there is zero need for those imports.

> Also templates were _always_ angry, so that was partially why I did it.

Not a valid reason. IDE is dumb. You are not.